subgraph/src/Stream.ts

```diff
@@ -0,0 +1,23 @@
+import { log } from "@graphprotocol/graph-ts";
+import { TokensWithdrawn, StreamCancelled } from "../generated/templates/Stream/Stream";
+import { Stream, Withdrawal } from "../generated/schema";
+
+export function handleTokensWithdrawn(event: TokensWithdrawn): void {
+  let withdrawal = new Withdrawal(event.transaction.hash);
```

Comment:
I think subgraph recommend using something like:
`event.transaction.hash.toHex() + "-" + event.logIndex.toString()`
wdyt?

Reply:
you're right; it's meant to prevent collisions in case multiple events are fired in the same blockchain tx
will fix